addons/auth/package.json

```diff
@@ -35,7 +35,6 @@
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
     "broccoli-asset-rev": "^3.0.0",
-    "core": "*",
```

Comment:
Mirage provided server mocking that we depended on.

Reply:
I think my question was more why was it coming from the package `core`? Am I misunderstanding this?

Reply:
Because core contains the api addon, which is where mirage lives.  Arguably I could have just imported api directly, but it would still ultimately be circular and the idea was always to completely decouple api and auth.